src/vs/platform/issue/common/issue.ts

```diff
@@ -42,7 +43,25 @@ export interface IssueReporterData {
 	issueType?: IssueType;
 }
 
+export interface ISettingSearchResult {
+	extensionId: string;
+	key: string;
+	score: number;
+}
+
+export interface ISettingsSearchIssueReporterData extends IssueReporterData {
+	issueType: IssueType.SettingsSearchIssue;
+	actualSearchResults: ISettingSearchResult[];
+	query: string;
+	filterResultCount: number;
+}
+
 export interface IIssueService {
```

Comment:
How about naming the new interface something like `IWorkbenchIssueService` and keeping the old one as `IIssueService`? To me, raw sounds like something that would always be wrapped/used as a base, but this is being used directly in the main process

Reply:
"Raw" is used in similar situations for search and debug but I'm not sure why... I like IWorkbenchIssueService better.